README.md

```diff
@@ -161,23 +161,20 @@ The method returns a dict object mapping a device type to a serialized device sp
 to instantiate a `Device` instance in the `Pulser` library.
 
 
-### Target different API endpoints
+### Using `env` parameter to target different API endpoints
 
 This is intended to the package developers.
```

Comment:
This is intended to the package developers or users which were given access to non-prod environments of the PASQAL cloud platform.